Automattic/pocket-casts-android
# End of Year: Sync the missing listening history

## Description

The change checks if there is any missing listening history at the start of the End of Year flow. If you have lots of missing listening history this might take awhile. I will take a look at the speed of this in a future change.

## Testing Instructions

1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in `base.gradle`
2. Run the `debugProd` build variant 
3. Login to your account which will sync 100 listening history episodes
4. When the prompt appears, tap "View My 2022"
5. The app will download all your episode listening history for 2022
6. ✅ Check the results seem correct
